doc/UsersGuide/source/FAQ.rst

```diff
@@ -8,136 +8,132 @@ FAQ
+
+      .. code-block:: console 
+
+         Job 7430255 finished for user Joe.Schmoe in partition hera with exit code 0:0
+
+      There is also a RESOURCE STATISTICS summary at the end of the test's ``out`` file. Errors will appear in the ``err`` file. Users can find log files with more detailed information in ``ufs-weather-model/tests/logs/log_<platform>`` (where platform is the name of the machine the user is running on, e.g., ``log_hercules``).
+   
+   #. When the build and run are complete, users can modify the namelist or ``model_configure`` files in the run directory (``${STMP}``) 
+      and re-run their forecast/test with modifications by submitting the ``job_card`` file:
 
       .. code-block:: console
 
          qsub job_card
+         # OR
+         sbatch job_card
 
 ============================================
 How do I change the length of the model run?
 ============================================
-In your run directory, there is a file named ``model_configure``.  Change the
-variable ``nhours_fcst`` to the desired number of hours.
+For individual RT tests, users can add the ``FHMAX`` variable to the test configuration file. For example, in the :wm-repo:`control_c48 <blob/develop/tests/tests/control_c48>` case, 
+users can increase the forecast duration from the default (``DAYS*24`` --- or 24 hours, in this case) to 48 hours by adding the statement:
+
+.. code-block:: console
+
+   export FHMAX=48
+
+Alternatively, users can set a different default value in :wm-repo:`default_vars.sh <blob/develop/tests/default_vars.sh>` by changing the ``FHMAX`` variable directly in ``default_vars.sh``
+or they can modify the ``nhours_fcst`` variable in the ``model_configure*`` file that their experiment uses. 
+
+To rerun a previously run test with a different forecast length, go to the run directory (usually ``${STMP}/FV3_RT/rt_PID/<test_name>``), and open the file named ``model_configure``.  
+Change the variable ``nhours_fcst`` to the desired number of hours for the forecast and rerun by sumbitting the job card, as described above.
 
 ==============================================================
 How do I set the output history interval?
 ==============================================================
 
-The interval at which output (history) files are written is controlled in two
-places, and depends on whether you are using the write component to generate your output files.
-:numref:`Table %s <OutputControl>` describes the relevant variables.  If the write_component is used, then the variables listed as ``model_configure`` are required.  It is however, also required that the settings in ``input.nml`` match those same settings in ``model_configure``.  If these settings are inconsistent, then unpredictable output files and intervals may occur!
+The interval at which output (history) files are written is controlled via the ``model_configure*`` files. 
```

Comment:
@mkavulich Here is the FAQ update you requested; we'd like to add it to the WM release docs. Please let me know if there are any issues/corrections.